packages/hash/frontend/src/components/Avatar.tsx

```diff
@@ -0,0 +1,40 @@
+import { VFC } from "react";
+import { Box, BoxProps } from "@mui/material";
+
+interface AvatarProps extends BoxProps {
+  title: string;
+  size?: number;
+}
+
+export const Avatar: VFC<AvatarProps> = ({ title, size = 32, ...props }) => {
+  const { sx, ...otherProps } = props;
+  return (
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      sx={{
+        width: size,
+        height: size,
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        borderRadius: "50%",
+        backgroundColor: ({ palette }) => palette.blue[70],
+        ...sx,
+      }}
+      {...otherProps}
+    >
+      <Box
+        component="span"
+        sx={{
+          color: ({ palette }) => palette.common.white,
+          fontSize: size / 2,
+          lineHeight: 1,
+        }}
+      >
+        {title?.charAt(0).toUpperCase()}
```

Comment:
```
{title.charAt(0).toUpperCase()}
```

Reply:
That wouldn't account for `title = ""` because `??` only checks for `undefined` or `null`

Reply:
Sorry just saw you updated this. I think in practice it should never be `""` because we require users to have a `preferredName` (which I assume is what we're passing here?). But we can still account for invalid props using `"U"`, so I'm happy with that approach.